tools/flang2/flang2exe/llutil.cpp

```diff
@@ -54,6 +54,54 @@ typedef struct LLDEF {
   struct LLDEF *next;
 } LLDEF;
 
+static const char *openmp_functions[] = {
+  "omp_set_dynamic",
+  "omp_get_dynamic",
+  "omp_set_num_threads",
+  "omp_get_max_threads",
+  "omp_get_level",
+  "omp_get_active_level",
+  "omp_in_parallel",
+  "omp_get_schedule",
+  "omp_set_schedule",
+  "omp_get_ancestor_thread_num",
+  "omp_get_thread_num",
+  "omp_get_team_size",
+  "omp_get_num_threads",
+  "omp_get_thread_limit",
+  "omp_get_num_procs",
+  "omp_set_nested",
+  "omp_get_nested",
+  "omp_set_max_active_levels",
+  "omp_get_max_active_levels",
+  "omp_get_proc_bind",
+  "omp_get_num_places",
+  "omp_get_place_num_procs",
+  "omp_get_place_proc_ids",
+  "omp_get_place_num",
+  "omp_get_partition_num_places",
+  "omp_get_partition_place_nums",
+  "omp_get_cancellation",
+  "omp_set_default_device",
+  "omp_get_default_device",
+  "omp_get_num_devices",
+  "omp_get_device_num",
+  "omp_get_num_teams",
+  "omp_get_team_num",
+  "omp_get_initial_device",
+  "omp_is_initial_device",
+  "omp_get_num_threads"
+};
+
+static bool is_openmp_function(const char *name) {
+  const int num_items = sizeof(openmp_functions)/sizeof(char *);
+  for (int i = 0; i < num_items; ++i) {
```

Comment:
Might consider adding a check if the function begins with omp_  (and return false early if not) before the loop.